Makefile.toml

```diff
@@ -234,14 +262,66 @@ alias = "build"
 [tasks.preview-docs]
 script = ['tools/gen-docs.sh']
 
+# Builds a local repository based on the 'latest' built targets.  Uses pubsys
+# to create a repo under /repo, named after the arch/variant/version,
+# containing subdirectories for the repo metadata and targets.
+[tasks.repo]
+dependencies = ["build"]
+script_runner = "bash"
+script = [
+'''
```

Comment:
There's a lot of data that pubsys needs from cargo-make and its environment.  docker-go uses arg parsing, but I think that'd be overwhelming and unhelpful with pubsys; would you recommend assuming that environment variables are set?  I worry about that too, because if it's in Makefile.toml directly, at least it's trivial to check if the variables are set.